packages/switchToggle/src/Default/styles.module.scss

```diff
@@ -0,0 +1,89 @@
+@use '@owlui/theme/src' as theme;
```

Comment:
Let's stick to the Bootstrap default styles and just add a different background to the `checked` input, any additional styles we get from Scott we can add afterwards. 

You can leave the styles here more like:
```scss
@use '@owlui/theme/src' as theme;

.switchToggle {
  $ref: &;

  &--appearance {
    &-default {
      &.theme--dark {
        .form-check-input {
          &:checked {
            background-color: var(--bs-dark);
          }
        }
      }
    }
  }

  &--size {
    &-lg {
      @include theme.typeface-lg;
    }

    &-md {
      @include theme.typeface-md;
    }

    &-sm {
      @include theme.typeface-sm;
    }
  }
}
```